src/services/hasura/hasura.service.ts

```
@@ -426,6 +432,35 @@ export class HasuraService {
       throw new HttpException('Bad request', HttpStatus.BAD_REQUEST);
     }
   }
+  async deleteJobsByBpps(bpps: string[]) {
+		// Build the _in clause for the query with the array of BPP IDs
+	const validBpps = bpps.filter((bpp) => bpp != null && bpp !== '');
+	if (validBpps.length === 0) {
+		this.logger.warn('All provided BPP IDs were invalid');
+		return { data: { [`delete_${this.cache_db}`]: { affected_rows: 0 } } };
+	}
+	const bppIdsArray = validBpps.map((bpp) => `"${bpp}"`).join(', ');
```

ESLint

[error] Expected indentation of 2 tabs but found 1.

(indent)